packages/core/security/core-security-server-internal/src/utils/default_implementation.ts

```diff
@@ -12,6 +12,7 @@ export const getDefaultSecurityImplementation = (): CoreSecurityDelegateContract
   return {
     authc: {
       getCurrentUser: () => null,
+      apiKeys: null,
```

Comment:
Does default implementation have to be null? 🤔 When will apiKeys service be null?

Reply:
See https://github.com/elastic/kibana/pull/186910#discussion_r1656982811

Reply:
But to answer the question: we have integration test suites where we boot core in oss mode (so we don't load the xpack plugins). This is only for those test/dev scenario that we had to add this stubbed "default implementation", for this very reason of avoiding having nullable/`?-able` APIs,